### PR TITLE
Display receipt analysis for newly uploaded receipt

### PR DESCRIPTION
**Pull request by avagarcia0**
* Read URL parameters and extract information about the receipt
* Remove hardcoded values and display parameterized values instead

image

### CODE REVIEW

File: src/main/webapp/js/receipt-analysis.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// limitations under the License.
+
+/** Fetches receipt properties from the server and adds them to the page. */
+function loadReceiptAnalysis() {
+  const timestamp = new URLSearchParams(location.search).get('timestamp');
+  const storeName = new URLSearchParams(location.search).get('store');
+  const total = new URLSearchParams(location.search).get('price');
+  const categories =
+      new URLSearchParams(location.search).get('categories').split(',');
+  const imageUrl = new URLSearchParams(location.search).get('image-url');
```
Comment (basaam0) [inline src/main/webapp/js/receipt-analysis.js:22]: Adding an intermediate variable for the URL parameters would make this simpler.

File: src/main/webapp/js/receipt-analysis.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  document.getElementById('timestamp').innerText = `Upload Date: ${timestamp}`;
```
Comment (basaam0) [inline src/main/webapp/js/receipt-analysis.js:24]: In #35, the timestamp URL parameter is in milliseconds, so eventually it should be formatted as a date. The `toLocaleString()` method may help with this.

`const date = new Date(timestamp).toLocaleString('en-US;, {...})`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

File: src/main/webapp/js/receipt-analysis.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  document.getElementById('store-name').innerText = `Store Name: ${storeName}`;
+  document.getElementById('total').innerText = `Total Price: $${total}`;
```
Comment (emilyvera) [inline src/main/webapp/js/receipt-analysis.js:26]: Is there supposed to be two $$ here

Comment (avagarcia0) [reply src/main/webapp/js/receipt-analysis.js:27]: Yeah, the first one is the currency symbol. Would it be better to use `'Total Price: $' + total` instead?

Comment (emilyvera) [reply src/main/webapp/js/receipt-analysis.js:27]: Ohhh I see, nope that makes sense now :)

**Pull request merged**